src/main/java/bgu/spl/mics/application/messages/AgentsAvailableObject.java

```diff
@@ -0,0 +1,102 @@
+package bgu.spl.mics.application.messages;
+
+import java.util.List;
+
+public class AgentsAvailableObject {
+    private List<String> _agentsSerials;
+    private List<String> _agentsNames; //TODO maybe only agents, event with serial and future with names?
+    private int _missionDuration;
```

Comment:
I think this is fine. either way you should probably drop the comment

Reply:
it was for you to read (: i'll remove it